OP_laba2.py

```diff
@@ -0,0 +1,11 @@
+<<<<<<< HEAD
+
+x=float(input("X="))
+y=float(input("Y="))
+answer = (x*y<=0 and y-x-1>=0)
```

Comment:
```suggestion
answer = (x * y <= 0 and y - x - 1 >= 0)
```
Formatting problems throughout the code.